tx/textual/valuerenderer/timestamp.go

```diff
@@ -0,0 +1,58 @@
+package valuerenderer
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"google.golang.org/protobuf/reflect/protoreflect"
+	tspb "google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type timestampValueRenderer struct{}
+
+// NewTimestampValueRenderer returns a ValueRenderer for protocol buffer Timestamp messages.
+// It renders timestamps using the RFC 3339 format, always using UTC as the timezone.
+// Fractional seconds are only rendered if nonzero.
+func NewTimestampValueRenderer() ValueRenderer {
+	return timestampValueRenderer{}
+}
```

Comment:
Note that I'm starting a convention of exporting functions to create the various default renderer types (and simplifying unit tests accordingly). This is to allow custom renderers to reuse standard components for portions of their representation rather than having to reinvent the wheel.